assets/sass/05-blocks/quote/_style.scss

```diff
@@ -1,7 +1,7 @@
 .wp-block-quote {
-	border-left: var(--quote--border-width) solid var(--quote--border-color);
```

Comment:
I think we're removing all uses of these two variables, so they can be removed from `assets/sass/01-settings/global.scss` too.

Reply:
Yes.